monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/pages/APIs.js

```diff
@@ -41,15 +40,7 @@ export default function APIs() {
     const selectedNodeList = useSelector(state => state.nodeList);
 
     const retrieveApis = () => {
-        var nodeListQueryParams="";
-        selectedNodeList.filter(node => {
-            nodeListQueryParams = nodeListQueryParams.concat(node, '&nodes=')
-        })
```

Comment:
Why are we removing this logic?  Btw, this logic seems to have an issue. Since it's a filter and it's not explicitly returning a boolean value. This might be used for mapping purposes. We may have to validate the usage of this logic if we haven't done that.

Reply:
Since we were using this logic in all the pages I moved that logic to a common method in `HTTPClient.js`